ocs_ci/framework/pytest_customization/ocscilib.py

```diff
@@ -432,6 +441,14 @@ def pytest_configure(config):
             check_clusters()
             if ocsci_config.RUN.get("cephcluster"):
                 gather_version_info_for_report(config)
+            if ocsci_config.MULTICLUSTER["acm_cluster"]:
+                config._metadata["ACM Operator"] = get_acm_version()
+                config._metadata["OCP DR Hub Operator"] = get_dr_hub_operator_version()
+                config._metadata[
+                    "ODF Multicluster Orchestrator"
+                ] = get_odf_multicluster_orchestrator_version()
+                config._metadata["GitOps Operator"] = get_ocp_gitops_operator_version()
```

Comment:
Are those both on RDR and MDR setup as well?
If not we will need to have some conditions.

Reply:
I think the RDR and MDR are the same for versions @sidhant-agrawal correct?